crates/full-node/sov-db/src/commit_flag.rs

```diff
@@ -120,80 +127,12 @@ impl CommitFlag {
         Ok(())
     }
 
-    pub fn log_reset_instruction(&self) {
-        tracing::error!(
-            "To reset commit flag, please remove commit flag file: `rm {}`",
```

Comment:
This was useful if user knows that it just wants to remove flag, they can just copy command and execute it. Why we removed it?